src/types/venues.ts

```diff
@@ -56,6 +58,7 @@ export type AnyVenue =
   | EmbeddableVenue
   | JazzbarVenue
   | PartyMapVenue
+  | AnimateMapVenue
   | PosterPageVenue;
```

Comment:
These are in alphabetic order (except for `GenericVenue` which is always at the top)
```suggestion
  | AnimateMapVenue
  | EmbeddableVenue
  | JazzbarVenue
  | PartyMapVenue
  | PosterPageVenue;
```